src/cluster_legacy.c

```diff
@@ -3926,6 +3952,11 @@ void clusterSetGossipEntry(clusterMsg *hdr, int i, clusterNode *n) {
 /* Send a PING or PONG packet to the specified node, making sure to add enough
  * gossip information. */
 void clusterSendPing(clusterLink *link, int type) {
+    serverLog(LL_DEBUG, "Sending %s packet to node %.40s (%s)",
+              clusterGetMessageTypeString(type),
+              link->node ? link->node->name : "<unknown>",
+              link->inbound ? "inbound" : "outbound");
```

Comment:
This is hundreds of messages per second being written into the log for large clusters. I'm not sure this is necessarily a good idea, as we sometimes enable this on production systems for debugging.

Reply:
`%.40s (%s)` is also usually the convention for the human readable nodename in (), which isn't true here, you are using inbound/outbound.

Reply:
We also log on every received cluster packet at the debug level, so I thought it wouldn't be much of a difference. And I added it because it was useful to me to debug this issue. But I can remove it. If I leave it, I can update to add the human readable nodename.